controller/backup_volume_controller.go

```diff
@@ -240,13 +237,14 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 			}
 			defer engineClientProxy.Close()
 
-			if err := backupTargetClient.BackupVolumeDelete(backupTargetClient.URL, backupVolumeName, backupTargetClient.Credential); err != nil {
+			if err := backupTargetClient.BackupVolumeDelete(backupTargetClient.URL, sourceBackupVolumeName, backupTargetClient.Credential); err != nil {
```

Comment:
OK, I think we can `canonicalBackupVolumeName` as I mentioned in another comment.
Let's unify the naming.

Reply:
Modified. Thanks